### PR TITLE
Branch 1

### PR DESCRIPTION
**Pull request by hoangtan15081995**

### CODE REVIEW

File: OPERATORS.HTML [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <p id="z6"></p>
+  <script>
+    let x = 6;
+    let y = 4;
+    let z1 = x + y;
+    let z2 = x - y;
+    let z3 = x * y;
+    let z4 = x % y;
+    let z5 = ++x;
+    let z6 = --x;
```
Comment (nvh0412) [inline OPERATORS.HTML]: why did we use `let` instead of `const`?

File: VARIABLES.HTML [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+</head>
+<body>
+  <h1> JavaScript Variables </h1>
+  <p id="demo1"></p>
+  <p id="demo2"></p>
+  <p id="demo3"></p>
+  <script>
+    let x = 4;
+    let y = 5;
+    let z = 6;
```
Comment (nvh0412) [inline VARIABLES.HTML:17]: here as well

Comment (hoangtan15081995) [reply VARIABLES.HTML:17]: ok thks bro

File: OPERATORS.HTML (hunk 1 above)
Comment (hoangtan15081995) [reply OPERATORS.HTML]: I think we can use both

**Review by nvh0412 — APPROVED** [review]

**Pull request merged**